### DISCUSSION TITLE
New features for Phase 4

### DISCUSSION THREAD

**Discussion by jashgopani** [jashgopani/application-tracking-system#12] [Polls]
Hi team,
Which of the following improvements do you think we should target for the Phase 3 release?

**Comment by rohanajm7** [comment]
I would say dockerizing the application would be much better since we have faced many issues with the setup. However, Refactoring the frontend code would also be a good feature to implement in our project. But, I feel that it is a large liability for this release.

**Reply by neerua08** [reply]
I also agree with this. I struggled to get the database running for a bit. A docker container that takes care of this can help resolve this issue.
